helm/adelphi/templates/schema-job.yaml

```diff
@@ -20,6 +20,9 @@ spec:
           echo "Configuring schema on the target cluster...";
           cat /schema/schema.cql | kubectl exec -n {{ .Values.namespace }} -i -t -c cassandra {{ .Values.target.clusterName }}-{{ .Values.target.dc }}-default-sts-0 -- /opt/cassandra/bin/cqlsh -u {{ .Values.target.clusterName }}-superuser -p $(cat /secret-target/password);
           echo "Done";
+          pip install cassandra-driver;
+          wget -q https://raw.githubusercontent.com/datastax/adelphi/master/schema-anonymizer/anonymizer.py;
```

Comment:
This setup will also be removed once most of this installation logic is converted to the Docker images